.vscode/settings.json

```diff
@@ -25,7 +25,15 @@
     "**/.cache/**": true,
     "**/temporian.egg-info/**": true,
   },
-  "python.formatting.provider": "black",
+  "black-formatter.interpreter": [
+    "${workspaceFolder}/.venv/bin/python"
```

Comment:
What about adding the instructions to setup the workspace (virtual-env?) for this to work?

Reply:
they were in the CONTRIBUTING guide already but as an optional step, reworded to make it mandatory